Dockerfile

```diff
@@ -48,7 +48,8 @@ COPY ./container/root /
 RUN tar xzf /tmp/s6-overlay-amd64.tar.gz -C / && \
     rm /tmp/s6-overlay-amd64.tar.gz
 
-EXPOSE 80
+# Using a non-privileged port to prevent having to use setcap internally
+EXPOSE 8080
```

Comment:
Do we need to modify this in Flight Director to make sure it matches this new port? across whichever apps are running from this parent container?